backend/src/main/java/com/github/wekaito/backend/CardService.java

```diff
@@ -93,44 +94,45 @@ void fetchCards() {
         List<Card> cards = new ArrayList<>();
 
         assert fetchedCards != null;
+        
         fetchedCards.forEach(card -> {
-
-            List<DigivolveCondition> digivolveConditions = card.digivolveCondition().stream()
-                    .map(condition -> new DigivolveCondition(
-                            condition.color(),
-                            Integer.parseInt(condition.cost()),
-                            Integer.parseInt(condition.level())
-                    ))
-                    .toList();
-
-            List<String> digiTypes = Arrays.stream(card.type().split("/")).toList();
-            List<String> colors = Arrays.stream(card.color().split("/")).toList();
-
-            cards.add(new Card(
-                    card.id(),
-                    card.name().english(),
-                    baseUrl + card.cardImage(),
-                    card.cardType(),
-                    colors,
-                    (card.attribute().equals("-")) ? null : card.attribute(),
-                    (card.cardNumber().equals("-")) ? null : card.cardNumber(),
-                    digivolveConditions,
-                    (card.specialDigivolve().equals("-")) ? null : card.specialDigivolve(),
-                    (card.form().equals("-")) ? null : card.form(),
-                    digiTypes,
-                    (card.dp().equals("-")) ? null : Integer.parseInt(card.dp()),
-                    (card.playCost().equals("-")) ? null : Integer.parseInt(card.playCost()),
-                    (card.cardLv().equals("-")) ? null : Integer.parseInt(card.cardLv().split("\\.")[1]),
-                    (card.effect().equals("-")) ? null : card.effect(),
-                    (card.digivolveEffect().equals("-")) ? null : card.digivolveEffect(),
-                    (card.aceEffect().equals("-")) ? null : card.aceEffect(),
-                    (card.burstDigivolve().equals("-")) ? null : card.burstDigivolve(),
-                    (card.digiXros().equals("-")) ? null : card.digiXros(),
-                    (card.dnaDigivolve().equals("-")) ? null : card.dnaDigivolve(),
-                    (card.securityEffect().equals("-")) ? null : card.securityEffect(),
-                    card.restrictions(),
-                    card.illustrator()));
+        List<DigivolveCondition> digivolveConditions = card.digivolveCondition().stream()
+                .map(condition -> new DigivolveCondition(
+                        condition.color(),
+                        Integer.parseInt(condition.cost()),
+                        !condition.level().equals("Tamer") ? Integer.parseInt(condition.level()) : null
```

Comment:
Is this a workaround or a proper solution?

Reply:
The best would be to make sure that the field is numeric.
Having said that, the only value that causes the issue in digivolution cost is "Tamer".